openedx/features/caliper_tracking/transformers/course_transformers.py

```diff
@@ -0,0 +1,15 @@
+"""
+Transformers for all course related events.
+"""
+
+
+def edx_course_enrollment_activated(current_event, caliper_event):
+    caliper_event['ip'] = current_event['ip']
+    if 'ip' in caliper_event['actor']:
+        del caliper_event['actor']['ip']
```

Comment:
already removed from base transformer, please remove this check

Reply:
ok